src/Helper/EntriesExport.php

```diff
@@ -432,6 +433,34 @@ private function produceTxt()
         );
     }
 
+    /**
+     * Dump a Markdown file.
+     *
+     * @return Response
+     */
+    private function produceMd()
+    {
+        $user = $this->tokenStorage->getToken() ? $this->tokenStorage->getToken()->getUser() : null;
+        \assert($user instanceof User);
+
+        $content = '';
+        $converter = new HtmlConverter();
+        $converter->getConfig()->setOption('strip_tags', true);
+        foreach ($this->entries as $entry) {
```

Comment:
Does it make sense to support multiple entries for markdown export? I guess users will expect multiple md files, not all in a single one

Reply:
Why not, but we need to rewrite the Export system. That's how we handle other exports.